src/app/sections/hero-section.tsx

```diff
@@ -1,73 +1,71 @@
 import React from "react";
 import homeData from "@/content/home.json";
 import Image from "next/image";
-import NavBar from "../components/NavBar";
+import NavBar from "../components/nav-bar";
+
 export default function HeroSection() {
   const { hero } = homeData;
   return (
-    <div className="flex flex-col bg-gradient-to-r from-[#9ec5f7] to-darkblue">
+    <div className="bg-gradient-to-r from-[#9ec5f7] to-darkblue">
       <NavBar />
-      <div className="lg:max-w-7xl lg:flex-row md:justify-between gap-4 py-10 flex flex-col">
-        <div className="lg:w-full lg:pl-20 lg:p-0 md:p-10 p-3 text-center">
+      <div className="flex flex-col gap-4 py-10 md:justify-between lg:flex-row lg:max-w-7xl lg:pb-5">
+        {/* Left Section */}
+        <div className="w-full p-3 text-center md:p-10 lg:w-[60%] lg:p-0 lg:pl-20">
           <div className="relative">
-            <div className="z-10 py-30">
-              <div className="text-white font-bold md:text-5xl lg:text-7xl lg:text-left text-3xl mb-7 text-center">
-                <h1>{hero.headline} </h1>
+            <div className="z-10 pt-40 lg:pt-20">
+              <div className="mb-7 text-3xl font-bold text-white text-center md:mt-30 md:text-5xl lg:text-left lg:text-7xl">
+                <h1>{hero.headline}</h1>
               </div>
-              <div className="text-center text-white lg:text-5xl lg:text-left md:text-2xl text-xl">
-                <span className="in-line">
-                  <h3> {hero.subheadline} </h3>
-                </span>
-                <br />
-                <span>
-                  <h3>{hero.subheadline2} </h3>
+              <div className="text-xl text-white text-center md:text-2xl lg:text-left lg:text-5xl">
+                <span className="inline-block">
+                  <h3>{hero.subheadline}</h3>
                 </span>
               </div>
             </div>
-            <div className="w-full absolute top-0 left-0 aspect-square">
+            <div className="absolute top-0 left-0 w-full aspect-square lg:w-[80%]">
               <Image
-                src="/images/Slide1.png"
-                alt="Slide 1"
+                src={hero.images.leftEffect}
+                alt="Left Section Effect"
```

Comment:
"Left" is not left anymore on mobile screen, right? Also, users that rely on screen readers wouldn't tell what is "left section". Try a different alt text.

```suggestion
                alt="Text's decorative effect"
```